src/frontend/src/utils/iiConnection.ts

```diff
@@ -161,8 +174,42 @@ export class IIConnection {
     };
   }
 
-  static async lookup(userNumber: UserNumber): Promise<DeviceData[]> {
-    return baseActor.lookup(userNumber);
+  static async fromSeedPhrase(
+    userNumber: bigint,
+    seedPhrase: string
+  ): Promise<LoginResult> {
+    const identity = await fromMnemonicWithoutValidation(
+      seedPhrase,
+      IC_DERIVATION_PATH
+    );
+    const delegationIdentity = await requestFEDelegation(identity);
```

Comment:
As discussed in Zoom, this may not be necessary. But it also does not hurt, so if it makes the code easier it can stay here.